client/src/app/all-projects/all-projects.component.ts

```diff
@@ -23,7 +20,9 @@ export class AllProjectsComponent implements OnInit, OnDestroy {
     ProjectColumn.short_name,
     ProjectColumn.project_title,
     ProjectColumn.primary_contributor,
-    ProjectColumn.last_updated
+    ProjectColumn.last_updated,
+    ProjectColumn.primary_wrangler,
+    ProjectColumn.wrangling_status
```

Comment:
Should be `wrangling_state` right. Also camelCase is preferred in JS

Reply:
yup, thanks! i've renamed them!